src/lib/components/Repeat/Daily/index.js

```diff
@@ -8,15 +8,22 @@ const RepeatDaily = ({
   },
   handleChange,
 }) => (
-  <div>
-    every
-    <input
-      name="repeat.daily.interval"
-      className="form-control"
-      value={interval}
-      onChange={numericalFieldHandler(handleChange)}
-    />
-    day(s)
+  <div className="form-group row d-flex align-items-sm-center">
+    <div className="col-sm-2" />
+    <div className="col-sm-1">
```

Comment:
if you want to move this element on the side use `.col-sm-offset-2` instead of putting empty div with `.col-sm-2`

Here and in other places.

Reply:
Check this out:
https://getbootstrap.com/docs/4.0/layout/grid/#offsetting-columns
It seems like `.col-sm-offset-2` has been discontinued and using 
` <div className="col-sm-2" />` may be the right solution.

Reply:
What I meant is using `.offset-md-2` as it is described in the doc.

Reply:
https://github.com/reactstrap/reactstrap/issues/586

I must have been using wrong bootstrap version, after explicitly setting bootstrap in package.json to version `^4.0.0-beta.2` it started working.